index.js

```diff
@@ -25,47 +49,71 @@ function makeCopier(copyConfig, assetManager, { compact } = {}) {
 		filter: copyConfig.filter
 	});
 	let { fingerprint } = copyConfig;
-	let plugins = determinePlugins(compact, copyConfig);
+	let compactors = determineCompactors(compact, copyConfig);
 
 	return filepaths => {
 		return Promise.all([
 			(filepaths ? fileFinder.match(filepaths) : fileFinder.all()),
 			determineTargetDir(source, target)
 		]).then(([fileNames, targetDir]) => {
 			return processFiles(fileNames, {
-				assetManager, source, target, targetDir, plugins, fingerprint
+				assetManager, source, target, targetDir, compactors, fingerprint
 			});
 		});
 	};
 }
 
-function determinePlugins(compact, copyConfig) {
+/**
+ * Determine which compactors should be used
+ *
+ * @param {boolean | undefined} compact
+ * @param {Config} copyConfig
+ * @returns {CompactorMap}
+ */
+function determineCompactors(compact, copyConfig) {
```

Comment:
🤔 Doesn't the logic here boil down to `(compact && copyConfig.compact) || {}` - in which case this function can be inlined, which arguably makes things more readable?

(I was trying to figure out in my head what the underlying logic was here; this trivial abstraction made that harder rather than easier.)

Reply:
Fair point! Fixed in 40771964f7d7d8bf54a2a06fe58275339ed1e485